buildcontext/excludes.go

```diff
@@ -10,23 +11,43 @@ import (
 
 // EarthIgnoreFile is the name of the earthly ignore file.
 const EarthIgnoreFile = ".earthignore"
+const EarthlyIgnoreFile = ".earthlyignore"
 
 // ImplicitExcludes is a list of implicit patterns to exclude.
 var ImplicitExcludes = []string{
 	".tmp-earthly-out/",
 	"build.earth",
 	"Earthfile",
 	EarthIgnoreFile,
+	EarthlyIgnoreFile,
 }
 
 func readExcludes(dir string) ([]string, error) {
-	filePath := filepath.Join(dir, EarthIgnoreFile)
+	var ignoreFile = EarthIgnoreFile
+
+	// if non (doesNotExist) errors appear then we need to return them.
```

Comment:
is this comment still relevant or can it be removed?

Reply:
Good spot - Removed and pushed.